scvid/module/onepass_mean_var_std.py

```diff
@@ -20,9 +23,19 @@ def __init__(self, transform: Optional[nn.Module] = None) -> None:
         self.x_squared_sums = 0
         self.x_size = 0
 
+    @staticmethod
+    def _get_fn_args_from_batch(
+        tensor_dict: Dict[str, torch.Tensor]
+    ) -> Tuple[Iterable, dict]:
+        x = tensor_dict["X"]
+        return (x,), {}
+
     def forward(self, x_ng: torch.Tensor) -> None:
         if self.transform is not None:
             x_ng = self.transform(x_ng)
+        num_replicas = get_rank_and_num_replicas()[1]
```

Comment:
Consider this instead:
```
_, num_replicas = get_rank_and_num_replicas()
```

The benefit is that you're additionally asserting that `get_rank_and_num_replicas()` returns a 2-tuple; more implicit or explicit asserts = more code safety!